### DISCUSSION TITLE
Graceful node shutdown

### DISCUSSION THREAD

**Discussion by xhejtman** [rancher/rke2#4395] [General]
Hello,

is it possible to setup [graceful node shutdown](https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown) with rke2? It seems that kubelet does not have cmdline options for this, only config is supported.

**Comment by brandond** [comment]
There's a thread about this in the k3s discussions, the same steps should work for RKE2.
https://github.com/k3s-io/k3s/discussions/4319#discussioncomment-1710171

**Comment by xhejtman** [comment]
Thank you, worked.
